.github/workflows/on-push-main.yml

```diff
@@ -90,3 +92,14 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npx --package=node@v18-lts -- "npx semantic-release"
+
+      - name: Generate JSON SBOM 🏛️
+        uses: CycloneDX/gh-node-module-generatebom@v1
+        with:
+          output: sbom.json
+
+      - name: Commit SBOM 👈
+        uses: EndBug/add-and-commit@v9
```

Comment:
fyi: If action/checkout is used, you don't need an external action to commit something:
https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action/58393457#58393457

Reply:
ok thanks! please check again.